src/main/java/com/ksm/robolo/roboloapp/domain/UserEntity.java

```diff
@@ -15,25 +16,24 @@
 	@GeneratedValue(strategy = GenerationType.AUTO)
 	private UUID id;
 
+	@Column(unique = true)
 	@NotEmpty(message = "Please provide your username")
-    @NotNull
+	@Length(min = 2, max = 50, message = "The username must be between 2 and 20 characters")
```

Comment:
gdzie wartość message będzie wyswietlana ,używana ?

Reply:
Jako message wyjątku, który poleci jeśli wymagania nie zostaną spełnione. To będzie ConstraintsViolationException czy jakoś tak. On jest łapany w serwisie, tylko w taki trochę dziwny sposób, bo on jest opakowany jeszcze przez kilka wyjątków i trzeba się do niego dobrać przechodząc przez stację tracę